src/main/java/uk/co/evoco/webdriver/configuration/WebDriverConfig.java

```diff
@@ -167,15 +157,13 @@ public void setBrowserPreferences(Map<String, ObjectNode> browserPreferences) {
     }
 
     /**
-     *
      * @return the run type
      */
     public RunType getRunType() {
         return runType;
     }
 
     /**
-     *
```

Comment:
These need adding back in - javadocs convention is to have a blank line first which is where the description goes